src/test/resources/regressions/features/termination/non-terminated_functions.gobra

```diff
@@ -0,0 +1,130 @@
+package termination
```

Comment:
Test files should have the following license header in the beginning of the file (before the package declaration):
```
// Any copyright is dedicated to the Public Domain.
// http://creativecommons.org/publicdomain/zero/1.0/
```
Also, test files should be properly formatted, otherwise they become very hard to read.